src/microbe_stage/PatchMapGenerator.cs

```diff
@@ -13,70 +13,79 @@ public static PatchMap Generate(WorldGenerationSettings settings, Species defaul
         var map = new PatchMap();
 
         // Predefined patches
-        Patch patch0 = new Patch("Pangonian vents", 0, GetBiomeTemplate("aavolcanic_vent"));
+        Patch patch0 = new Patch(TranslationServer.Translate("PATCH_PANGONIAN_VENTS"), 0,
+            GetBiomeTemplate("aavolcanic_vent"));
         patch0.Depth[0] = 2500;
         patch0.Depth[1] = 3000;
         patch0.AddSpecies(defaultSpecies);
         patch0.ScreenCoordinates = new Vector2(100, 400);
         map.AddPatch(patch0);
 
-        Patch patch1 = new Patch("Pangonian Mesopelagic", 1, GetBiomeTemplate("mesopelagic"));
+        Patch patch1 = new Patch(TranslationServer.Translate("PATCH_MESOPELAGIC"), 1,
```

Comment:
All of these patch names as these are Pangonian (which is just a random place that was thought up) specific locations should include that part in the name like PATCH_PANGONIAN_VENTS does.